modules/rosa-sdpolicy-networking.adoc

```diff
@@ -100,6 +100,3 @@ For {product-title} clusters that have a private cloud network configuration, a
 Network verification checks run automatically when you deploy a {product-title} cluster into an existing Virtual Private Cloud (VPC) or create an additional machine pool with a subnet that is new to your cluster. The checks validate your network configuration and highlight errors, enabling you to resolve configuration issues prior to deployment.
 
 You can also run the network verification checks manually to validate the configuration for an existing cluster.
-ifeval::["{context}" == "rosa-hcp-service-definition"]
```

Comment:
Typically these are needed so run-on rendering issues do not occur to files that follow. Maybe add back this statement bug leave a blank line before it. This might fix the rendering issue.

Setting and unsetting a temporary attribute is outlined in our [OCP guidelines|https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#conditionals-for-different-assemblies]